src/driver/CMake/config/dkms.conf.in

```diff
@@ -1,9 +1,16 @@
 PACKAGE_NAME=@XDNA_DKMS_PKG_NAME@
 PACKAGE_VERSION="@XRT_PLUGIN_VERSION_STRING@"
 BUILD_EXCLUSIVE_KERNEL_MIN=6.8
-MAKE[0]="cd driver/@XDNA_DRV_DIR@; make @XDNA_DRV_BLD_FLAGS_DKMS@ KERNEL_SRC=${kernel_source_dir}; cd ../.."
-CLEAN="cd driver/@XDNA_DRV_DIR@; make clean KERNEL_SRC=${kernel_source_dir}; cd ../.."
+
+MAKE="cd driver/@XDNA_DRV_DIR@; make @XDNA_DRV_BLD_FLAGS_DKMS@ KERNEL_SRC=${kernel_source_dir}; cd ../@XDNA_DRV_PF_DIR@; make KERNEL_SRC=${kernel_source_dir}; cd ../.."
```

Comment:
this will remove building regular driver and only build the PF driver?
it seems a bug to me.

Reply:
I recall Min suggested MAKE[0] and MAKE[1]. But, if this common solution works better, then we should use this.
As long as, making pf driver or adding code into pf driver would not affect amdxdna aie2 driver make, this is good solution.